com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -465,7 +465,10 @@ private void Initialize(bool server)
             // Build the NetworkPrefabOverrideLinks dictionary
             for (int i = 0; i < NetworkConfig.NetworkPrefabs.Count; i++)
             {
-                if (NetworkConfig.NetworkPrefabs[i] == null || NetworkConfig.NetworkPrefabs[i].Prefab == null)
+                var sourcePrefabGlobalObjectIdHash = (uint)0;
```

Comment:
last but not least, I'd suggest to run `./standards.py --check` and maybe `./standards.py --fix` locally to check & fix spaces/namings locally — just in case :P